src/backend/partaj/core/models/referral.py

```diff
@@ -589,6 +589,36 @@ def change_urgencylevel(
             item_content_object=referral_urgencylevel_history,
         )
 
+        # Define all users who need to receive emails for this referral
+        targets = [self.user]
+        if self.assignees.count() > 0:
+            targets = targets + list(self.assignees.all())
+        else:
+            for unit in self.units.all():
+                targets = targets + [
+                    membership.user
+                    for membership in unit.get_memberships().filter(
+                        role=UnitMembershipRole.OWNER
+                    )
+                ]
+                targets = targets + [
+                    membership.user
+                    for membership in unit.get_memberships().filter(
+                        role=UnitMembershipRole.ADMIN
+                    )
+                ]
+
+        # delete double user
+        targets = set(targets)
+
+        for target in targets:
+            Mailer.send_referral_changeurgencylevel(
```

Comment:
As there are a lot of arguments you should use named arguments to make things as clear as possible